controllers/release/adapter.go

```diff
@@ -180,14 +181,58 @@ func (a *adapter) EnsureReleaseIsRunning() (controller.OperationResult, error) {
 	return controller.ContinueProcessing()
 }
 
+// EnsureTenantReleasePipelineIsProcessed is an operation that will ensure that a Tenant Release PipelineRun associated to the Release
+// being processed and a RoleBinding to grant its serviceAccount permissions exist. Otherwise, it will create them.
+func (a *adapter) EnsureReleaseTenantPipelineIsProcessed() (controller.OperationResult, error) {
+	if a.release.HasProcessingFinished() {
+		return controller.ContinueProcessing()
+	}
+
+	pipelineRun, err := a.loader.GetReleasePipelineRun(a.ctx, a.client, a.release)
+	if err != nil && !errors.IsNotFound(err) {
+		return controller.RequeueWithError(err)
+	}
+
+	roleBinding, _ := a.loader.GetRoleBindingFromReleaseStatus(a.ctx, a.client, a.release)
```

Comment:
We aren't passing the roleBinding in this operation, so I think we can remove this line and just pass nil in its place on line 222

Reply:
removed.